py/bitbox02/bitbox02/bitbox02/bitbox02.py

```diff
@@ -648,16 +648,6 @@ def insert_sdcard(self) -> None:
         )
         self._msg_query(request, expected_response="success")
 
-    def remove_sdcard(self) -> None:
```

Comment:
This implies a major version bump of the bitbox02 Python library (7.0.0 => 8.0.0) and a CHANGELOG entry to py/bitbox02/CHANGELOG.md